VideoLocker/src/test/java/org/edx/mobile/test/util/TimeUtilsForTests.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.test.util;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class TimeUtilsForTests {
+    private static String DEFAULT_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
+    private static SimpleDateFormat dateFormat = new SimpleDateFormat(DEFAULT_DATE_FORMAT, Locale.US);
+
+    public static final TimeZone DEFAULT_TIME_ZONE = TimeZone.getTimeZone("UTC");
+    public static final long DEFAULT_TIME = getMillis("2016-1-1 00:00:00");
```

Comment:
Hmm, these strings should have some comments explaining them, e.g.

// Define a constant time and time zone to get predictable test parameters.